http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/RuntimeServiceInterp.scala

```diff
@@ -115,7 +116,8 @@ class RuntimeServiceInterp[F[_]: Parallel](config: RuntimeServiceConfig,
                         .map(diskResult =>
                           RuntimeConfig.GceWithPdConfig(
                             gce.machineType.getOrElse(config.gceConfig.runtimeConfigDefaults.machineType),
-                            Some(diskResult.disk.id)
+                            Some(diskResult.disk.id),
+                            config.gceConfig.runtimeConfigDefaults.bootDiskSize.get // .get here is okay. We've verified the data in Config.scala
```

Comment:
I guess it's a little unfortunate that Config `gce.runtimeDefaults` uses the `GceConfig` class so we have to deal with `Option` here. Maybe we should have a `GceConfigDefaults` type (not necessarily in this PR).

Reply:
yea...I created https://broadworkbench.atlassian.net/browse/IA-2014?atlOrigin=eyJpIjoiMTQ1YjBiMzJkNThiNDM1OWE5ODM0ZTE3YTQ0MjNkZTgiLCJwIjoiamlyYS1zbGFjay1pbnQifQ to address similar issue, will probably address this one too